tests/DI/EventDispatcherExtensionTest.php

```diff
@@ -39,6 +39,8 @@ public function testLoadSubscribers()
 		$containerBuilder->addDefinition('eventSubscriber')
 			->setClass(SomeEventSubscriber::class);
 
+		$containerBuilder->prepareClassList();
```

Comment:
Could you also add note about dependency on Compiler life cycle?
With link if possible, so it's obvious to other people too.